src/styles/index.scss

```diff
@@ -1,3 +1,154 @@
+@import './variables';
+
+// Міксин для центрованих елементів
+@mixin flex-center($width, $height) {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: $width;
+  height: $height;
+}
```

Comment:
Consider using more descriptive mixin names. The name `flex-center` is not fully descriptive of what the mixin does, as it also sets the width and height. A more descriptive name could be `flex-center-with-size` to reflect all its responsibilities.